queue_services/payment-reconciliations/src/reconciliations/payment_reconciliations.py

```diff
@@ -391,6 +391,7 @@ async def _process_paid_invoices(inv_references, row):
         logger.debug('PAID Invoice. Invoice Reference ID : %s, invoice ID : %s', inv_ref.id, inv_ref.invoice_id)
 
         inv.invoice_status_code = InvoiceStatus.PAID.value
+        inv.payment_date = receipt_date
```

Comment:
I think it's safe for us to just keep this as datetime.now(), that way we know exactly when it was set to PAID in our system.